src/main/java/org/spongepowered/api/stats/Statistics.java

```diff
@@ -0,0 +1,86 @@
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.spongepowered.api.stats;
+
+/**
+ * A utility class for getting all available {@link Statistic}s that are not
+ * grouped.
+ */
+public final class Statistics {
+
+    public static final Statistic ANIMALS_BRED = null;
+    public static final Statistic ARMOR_CLEANED = null;
+    public static final Statistic BANNER_CLEANED = null;
+    public static final Statistic BEACON_INTERACTION = null;
+    public static final Statistic BOAT_DISTANCE = null;
+    public static final Statistic BREWINGSTAND_INTERACTION = null;
+    public static final Statistic CAKE_SLICES_EATEN = null;
+    public static final Statistic CAULDRON_FILLED = null;
+    public static final Statistic CAULDRON_USED = null;
+    public static final Statistic CHEST_OPENED = null;
+    public static final Statistic CLIMB_DISTANCE = null;
+    public static final Statistic CRAFTING_TABLE_INTERACTION = null;
+    public static final Statistic CROUCH_DISTANCE = null;
+    public static final Statistic DAMAGE_DEALT = null;
+    public static final Statistic DAMAGE_TAKEN = null;
+    public static final Statistic DEATHS = null;
+    public static final Statistic DISPENSER_INSPECTED = null;
+    public static final Statistic DIVE_DISTANCE = null;
+    public static final Statistic DROP = null;
+    public static final Statistic DROPPER_INSPECTED = null;
+    public static final Statistic ENDERCHEST_OPENED = null;
+    public static final Statistic FALL_DISTANCE = null;
+    public static final Statistic FISH_CAUGHT = null;
+    public static final Statistic FLOWER_POTTED = null;
+    public static final Statistic FURNACE_INTERACTION = null;
+    public static final Statistic FLY_DISTANCE = null;
+    public static final Statistic HOPPER_INSPECTED = null;
+    public static final Statistic HORSE_DISTANCE = null;
+    public static final Statistic ITEM_ENCHANTED = null;
+    public static final Statistic NOTEBLOCK_PLAYED = null;
+    public static final Statistic NOTEBLOCK_TUNED = null;
+    public static final Statistic JUMP = null;
+    public static final Statistic JUNK_FISHED = null;
+    public static final Statistic LEAVE_GAME = null;
+    public static final Statistic MINECART_DISTANCE = null;
+    public static final Statistic MOB_KILLS = null;
+    public static final Statistic PIG_DISTANCE = null;
+    public static final Statistic PLAYER_KILLS = null;
+    public static final Statistic PLAY_ONE_MINUTE = null;
```

Comment:
This one should be renamed to `TIME_PLAYED`, because its measured in ticks and not in minutes.